.github/workflows/release.yml

```diff
@@ -1213,6 +1213,7 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
         prerelease: false
         allowUpdates: true
+        omitBody: true
```

Comment:
Just wondering, why not `omitBodyDuringUpdate`? I think I don't understand the workflow well enough

Reply:
Makes sense. Missed that in https://github.com/ncipollo/release-action

Updated